sdk/include/wdf/kmdf/1.9/wdfverifier.h

```diff
@@ -0,0 +1,83 @@
+/*
+ * wdfverifier.h
+ *
+ * Windows Driver Framework - Driver Framework Verifier definitioons
+ *
+ * This file is part of the ReactOS WDF package.
+ *
+ * Contributors:
+ *   Created by Benjamin Aerni <admin@bennottelling.com>
+ *
+ * Intended Usecase:
+ *   Kernel mode drivers
+ *
+ * THIS SOFTWARE IS NOT COPYRIGHTED
+ *
+ * This source code is offered for use in the public domain. You may
+ * use, modify or distribute it freely.
+ *
+ * This code is distributed in the hope that it will be useful but
+ * WITHOUT ANY WARRANTY. ALL WARRANTIES, EXPRESS OR IMPLIED ARE HEREBY
+ * DISCLAIMED. This includes but is not limited to warranties of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ */
+#ifndef _WDFVERIFIER_H_
+#define _WDFVERIFIER_H_
+
+#if (NTDDI_VERSION >= NTDDI_WIN2K)
+
+/* 
+WDF Function: WdfVerifierDbgBreakPoint
+*/
+typedef
+NTAPI
+WDFAPI
+VOID
+(*PFN_WDFVERIFIERDBGBREAKPOINT)
+(
+    __in PWDF_DRIVER_GLOBALS DriverGlobals
+);
+
+VOID
+FORCEINLINE
+WdfVerifierDbgBreakPoint
+(    )
```

Comment:
Wtf?

Reply:
This definition is equal to MS's, just with my (I suppose weird) formatting

Reply:
I would add a "VOID" inside.